packages/features/ee/teams/lib/removeMember.ts

```diff
@@ -101,16 +114,103 @@ const removeMember = async ({
       }),
       ProfileRepository.delete({
         userId: membership.userId,
-        organizationId: orgInfo.id,
+        organizationId: orgId,
       }),
     ]);
+
+    // delete all workflowReminders of the removed team member that come from org workflows
+    const workflowRemindersToDelete = await prisma.workflowReminder.findMany({
+      where: {
+        workflowStep: {
+          workflow: {
+            teamId: orgId,
+          },
+        },
+        booking: {
+          eventType: {
+            userId: memberId,
+          },
+        },
+      },
+      select: {
+        id: true,
+        referenceId: true,
+        method: true,
+      },
+    });
+
+    deleteAllWorkflowReminders(workflowRemindersToDelete);
   }
 
   // Deleted managed event types from this team from this member
   await prisma.eventType.deleteMany({
     where: { parent: { teamId: teamId }, userId: membership.userId },
   });
 
+  // cancel/delete all workflowReminders of the removed member that come from that team (org teams only)
+  if (team.parentId) {
+    const isUserMemberOfOtherTeams = !!foundUser.teams.filter(
+      (userTeam) => userTeam.team.id !== team.id && !!userTeam.team.parentId
+    ).length;
+
+    const removedWorkflows = await prisma.workflow.findMany({
+      where: {
+        OR: [
+          {
+            activeOnTeams: {
+              some: {
+                teamId: team.id,
+              },
+              //don't delete reminder, if user is still part of another team that is active on this workflow
+              none: {
+                team: {
+                  members: {
+                    some: {
+                      userId: memberId,
+                    },
+                  },
+                },
+              },
+            },
+            // if user is still a member of other teams in the org, we also need to make sure that the found workflow is not active on all teams
+            ...(isUserMemberOfOtherTeams && {
+              isActiveOnAll: false,
+            }),
+          },
+          {
+            // workflows of the org that are set active on all teams (and the user is not member of any other team)
```

Comment:
this case wasn't covered before: 

- if activeOnAll is true and user is not part of any other org team